src/main/drivers/bus_spi.h

```diff
@@ -67,39 +46,23 @@ typedef enum {
 
 typedef enum SPIDevice {
     SPIINVALID = -1,
+#if defined(USE_SPI_DEVICE_0)
+    SPIDEV_0   = 0,
+    SPIDEV_1,
+#else
     SPIDEV_1   = 0,
+#endif
```

Comment:
Maybe, to make it more future-proof: (ideally also using SPIDEV_FIRST when device index is necessary)
```suggestion
    SPIDEV_FIRST = 0,
#if defined(USE_SPI_DEVICE_0)
    SPIDEV_0   = SPIDEV_FIRST ,
    SPIDEV_1,
#else
    SPIDEV_1   = SPIDEV_FIRST,
#endif
```

Reply:
done :)